.prettierrc

```diff
@@ -0,0 +1,7 @@
+{
+	"printWidth": 100,
+	"singleQuote": true,
+	"useTabs": true,
```

Comment:
Why was this file added?

Reply:
I added eslint and prettier to have a well known code style

Reply:
This isn't a PR to change the code formatting though.

Reply:
> > What do you mean by that? The git stuff or the approach itself?
> 
> Make this PR only converting it to typescript, remove all the other changes. The first PR should be just convert all files to ts and enable loose checking. Second PR should be adding types. https://github.com/egoist/tsup should be used for bundling so we get the different bundle we need. It's this kind of things that would have been mentioned if an issue had been opened prior to this.
> 
> > therefore the git (rename) history is limited anyways.
> 
> Wrong. Go back and look at the rewrite I did from agenda to what it is now. This can be done properly.

I didn't expect a feedback like this, sorry if I somehow insulted you or something. 

It's an initial draft, we can go from here anywhere. I just did this because we use agenda, if you do not agree with my way, this is totally fine. I'm not saying we need to get this into this repo or anything like this. It's just an suggestion.
Code style or not, complete rewrite or not, any other approach (using tsup,.. plain tsc,..) or not,... are all things we can probably solve. If we want to and work together, but we do not need to. It's up to you and the community :) I just wanted to contribute my work, and maybe kick some things off.

Anyways, thanks for your feedback so far.